packages/runtime/src/provider.ts

```diff
@@ -34,17 +34,7 @@ function commandsForRuntime(name: string, entryPath: string): RuntimeCommands {
     return { run: `cat ${entryPath} | sqlite` };
   }
 
-  if (name === "clang") {
-    return {
-      prepare: [
-        `clang -cc1 -Werror -triple wasm32-unkown-wasi -isysroot /sys -internal-isystem /sys/include -ferror-limit 4 -fmessage-length 80 -fcolor-diagnostics -O2 -emit-obj -o ./program.o ${entryPath}`,
```

Comment:
Instead of removing the `clang` runtime I'd like to see `clang` here switched for `runno-clang`.